packages/cli/test/unit/commands/env/index.test.ts

```diff
@@ -1,5 +1,11 @@
-import { describe, it } from 'vitest';
+import { describe, it, expect } from 'vitest';
+import env from '../../../../src/commands/env';
+import { client } from '../../../mocks/client';
 
 describe('env', () => {
-  it.todo('errors when invoked without subcommand');
+  it('errors when invoked without subcommand', async () => {
+    client.setArgv('env');
+    const exitCodePromise = env(client);
+    expect(exitCodePromise).resolves.toBe(1);
```

Comment:
Interesting observation that this is `1` where the others following this pattern are `2`